timed/tracking/views.py

```diff
@@ -171,10 +171,11 @@ def update(self, request, *args, **kwargs):
             # value equal None means do not touch
             if value is not None
         }
-        if fields and request.user != instance.user:
-            tasks.notify_user_changed_report(instance, fields, request.user)
-        if fields.get("rejected"):
-            tasks.notify_user_rejected_report(instance, request.user)
```

Comment:
you could even include the lines L168-173 and add a comment on the reason why this is done to avoid head scratching and guessing next time someone reads those lines